### DISCUSSION TITLE
Isso aqui é muito complicado

### DISCUSSION THREAD

**Discussion by AstroSymas** [community/community#145266] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Essa parada é muito complicada, eu faço as coisas e nem sei ao menos oq estou fazendo.
É exaustivo, pois nem sei na verdade onde tudo isso vai dar, lq vai acontecer, é massante.
Essa site é estranho, não sei quem tá me vendo aqui, se são pessoas de verdade ou sei lá oq, se são pessoas ao meu redor...
Me sinto retardado as vezes, perdendo tempo com sei lá oq .

**Comment by NCatalani** [comment]
![image](https://github.com/user-attachments/assets/f73f9001-031f-41a0-b087-370fe5a4d0b8)

**Comment by shinybrightstar** [comment]
Hi @AstroSymas , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
